core/storage/app_feedback_report/gae_models.py

```diff
@@ -297,7 +324,7 @@ def export_data(cls, user_id):
         user_data = dict()
         report_models = cls.get_all().filter(
             cls.scrubbed_by == user_id).fetch()
-
+        report_models = cast(List[AppFeedbackReportModel], report_models)
```

Comment:
I've mentioned this before, but try not to redeclare variables with the same name. Cast inline instead.

Please fix this throughout your PR, not just here. If you don't fix the general comments then the reviewer will just ask you to do that again, and that wastes time.

Reply:
Yeah I missed a couple of these. Fixed them!